frontend/src/stores/OutputStore.js

```diff
@@ -228,6 +247,101 @@ class OutputStore {
                 break;
         }
     }
+
+    @observable plotData = [];
+    @action setPlotData() {
+        this.plotData = [];
+        let output = this.getCurrentOutput(this.selectedDatasetIndex);
+        if (output == null || "error" in output) {
+            return output;
+        }
+        let doses = output.dataset.doses;
+        let mean = output.dataset.means;
+        let stdevs = output.dataset.stdevs;
+        let ns = output.dataset.ns;
+        let errorbars = [];
+        for (var i = 0; i < stdevs.length; i++) {
+            var value = stdevs[i] / Math.sqrt(ns[i]);
+            errorbars.push(value);
+        }
+        var trace1 = {
+            x: doses,
+            y: mean,
+            error_y: {
+                type: "data",
+                array: errorbars,
+                visible: true,
+            },
+            mode: "markers",
+            type: "scatter",
+            name: "estimated probability",
+        };
+        this.plotData.push(trace1);
+    }
+    @observable layout = {
+        title: {
+            text: "",
+            font: {
+                family: "Courier New, monospace",
+                size: 12,
+            },
+            xref: "paper",
+            x: 0.05,
+        },
+        xaxis: {
+            title: {
+                text: "Dose (mg/kg-day)",
+                font: {
+                    family: "Courier New, monospace",
+                    size: 14,
+                    color: "#7f7f7f",
+                },
+            },
+        },
+        yaxis: {
+            title: {
+                text: "Response (mg/dL)",
+                font: {
+                    family: "Courier New, monospace",
+                    size: 14,
+                    color: "#7f7f7f",
+                },
+            },
+        },
+    };
+
+    @action addPlotData = index => {
+        let output = this.getCurrentOutput(this.selectedDatasetIndex);
+        let currentModel = output.models.find(item => item.model_index == index);
+        let cdf = currentModel.results.cdf;
+        let doses = output.dataset.doses;
+
+        let doseRange = _.max(doses) - _.min(doses);
+        let pValue = this.getPValue;
+        let xArray = [];
+        pValue.map(item => {
+            let new_item = (item / 100) * doseRange;
```

Comment:
so this should be the array of cdf, not pvalues. 

- for x, you use the index of the value (so b/w 0-99 and multiply by doseRange)
- for y, you'll calculate a responseRange just like dose-range that uses either response mean for continuous data or incidence/n for dichotomous data, and you'll multiple it by the value